docs/src/pages/components/about-the-lab/about-the-lab-ja.md

```diff
@@ -2,18 +2,18 @@
 
 <p class="description">このパッケージは、まだコアに移動する準備ができていないインキュベーター・コンポーネントをホストします。</p>
 
-Labパッケージとcoreパッケージの明確な違いはどのようにバージョン管理されるかです。 Labパッケージとcoreパッケージの明確な違いはどのようにバージョン管理されるかです。 Coreパッケージは[slower-moving policy](https://material-ui.com/versions/#release-frequency).
+Labパッケージとcoreパッケージの明確な違いはどのようにバージョン管理されるかです。 Labパッケージとcoreパッケージの明確な違いはどのようにバージョン管理されるかです。 Labパッケージとcoreパッケージの明確な違いはどのようにバージョン管理されるかです。 Coreパッケージは[slower-moving policy](https://material-ui.com/versions/#release-frequency).
```

Comment:
@mbrookes it's not working, I'm closing and pausing the sync

<img width="527" alt="Capture d’écran 2021-07-01 à 18 21 35" src="https://user-images.githubusercontent.com/3165635/124157586-3640cd00-da99-11eb-8a7f-d5c74629d556.png">